docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -163,21 +163,6 @@ class C
 }
 ```
 
-## Warn for redundant pattern in simple `or` patterns
```

Comment:
This flow is reverting changes. Perhaps we need the [forward flow](https://github.com/dotnet/dotnet/pull/3147) to be merged first.